src/main/java/com/fasterxml/jackson/databind/deser/std/StdKeyDeserializer.java

```diff
@@ -417,6 +430,20 @@ private EnumResolver _getToStringResolver(DeserializationContext ctxt)
             }
             return res;
         }
+
+        private EnumResolver _getIndexResolver(DeserializationContext ctxt) {
+            EnumResolver res = _byIndexResolver;
```

Comment:
I think it would be better to not have the `res` local variable and to refer to `_byIndexResolver ` directly throughout this method